README.md

```diff
@@ -81,6 +81,16 @@ to `libblp.dll` and ensure it ends up on the load PATH.
 By default, node-gyp compiles ffi for x64 so make sure `libblp.dll`
 matches this architecture.
 
+#### Ubuntu, Debian
+```shell
+$ sudo apt-get install cmake git gcc
+$ git clone git://github.com/Kanma/BLPConverter.git
+$ cd BLPConverter
+$ cmake CMakeLists.txt -DWITH_LIBRARY=YES
+$ sudo make install
+$ sudo ldconfig
+```
```

Comment:
Great to finally have this in place, thanks! It'll resolve issue #88.

Could you remove the $-indicator on each line so that it can be easily copy pasted?